website/templates/public/pages/help/licenses.mako

```diff
@@ -1,5 +1,5 @@
 <div class="col-sm-12">
     <span id="licenses" class="anchor"></span>
     <h4 class="m-t-lg f-w-lg">Licenses</h4>
-    <p>On a project's Overview page, you can add a license by clicking the &ldquo;No license&rdquo; text below the project's description.</p>
+    <p>You can add a license to your project to allow others to copy, distribute, and make use of your data while allowing you to retain copyright. On a project's Overview page, add a license by clicking the &ldquo;No license&rdquo; text below the project's description and choose from a selection of common license types or upload your own. Not sure what license to apply to your work? Get detailed information here:<a target="_blank" href="http://choosealicense.com/">http://choosealicense.com/</a></p>
```

Comment:
Ah, merged prematurely. A few things to fix here:

- Missing space after colon
- Clicking link to choosealicense.org doesn't work
- Missing period at end of sentence.